webapi/src/main/scala/org/knora/webapi/responders/v1/PermissionsResponderV1.scala

```diff
@@ -191,8 +191,12 @@ class PermissionsResponderV1 extends ResponderV1 {
 
         /* combines all permissions for each project and removes duplicate permissions inside a project  */
         val result: Future[Map[IRI, Set[PermissionV1]]] = for {
-            allPermission <- allPermissionsFuture
-            result = allPermission.groupBy(_._1).map { case (k, v) =>
+            allPermissions <- allPermissionsFuture
```

Comment:
Please add a type annotation for `allPermissions`. Tuples are hard to read even without type annotations.